.gitignore

```diff
@@ -3,4 +3,4 @@
 dist/
 build/
 launchable/__pycache__
-launchable/tests/__pycache__
+tests/__pycache__
```

Comment:
Add `__pycache__/` for other directories. Or https://github.com/github/gitignore/blob/master/Python.gitignore is much better

Reply:
That looks way much better! Thank you!